packages/flutter/lib/src/widgets/form.dart

```diff
@@ -162,7 +162,9 @@ class Form extends StatefulWidget {
 class FormState extends State<Form> {
   int _generation = 0;
   bool _hasInteractedByUser = false;
-  final Set<FormFieldState<dynamic>> _fields = <FormFieldState<dynamic>>{};
+
+  /// Can access [FormFieldState] in public
```

Comment:
It's not clear to me why any of these fields/methods should be public. Can you explain your use case?

Also, if we were to make these public, they need more detailed docs.

Reply:
Related to [https://github.com/flutter/flutter/issues/67283](#67283).

I want to functionality like when form validated then scroll to first `FormFieldState` in the Set that validate error occurred.

I don't know if flutter have the solution for this.

What do you think?

Reply:
if you have the globalkey to the individual formfield, you can also use that the retrieve the context to scroll.

Reply:
@chunhtai 

But when I create dynamic form from the api that very inconvenient because It need pass down the GlobalKey as a parameter to the widget. (In my case, it deep)